modules/ding_app_variables/ding_app_variables.module

```diff
@@ -28,7 +28,14 @@ function ding_app_variables_menu() {
  *   List of all variables to the output.
  */
 function ding_app_variables_display() {
-  $carousel_configs = variable_get('ting_carousel_search_queries');
+  $carousels = array(
+    'carousel',
+  );
```

Comment:
So we create an array with one element to loop over that one element in the next lines. Why not

```
$carousel_configs[] = _ding_app_variables_get_carousel_configs('carousel');
```